staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -103,7 +103,7 @@ var (
 			// This metric is used for verifying api call latencies SLO,
 			// as well as tracking regressions in this aspects.
 			// Thus we customize buckets significantly, to empower both usecases.
-			Buckets: []float64{0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3,
+			Buckets: []float64{0.005, 0.025, 0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3,
```

Comment:
IIRC, this change is for consistency purposes between both metrics, but even though 2 buckets isn't much when we are looking at a total of 23, it still results in a couple of thousand series which is not negligible. Since @wojtek-t has recently revisited the list I'd like his approval before proceeding.

Reply:
I personally don't think these will provide any useful value, but I'm also not opposed to them. We used to have ~40 buckets before, so going back to 23 doesn't sound completely terrible.